migrations/Version20240108140231.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
Consider running 
```
itkdev-docker-compose bin/console doctrine:migrations:execute --down DoctrineMigrations\\Version20240108140231
```
for each of the 3 migrations in the feature. Afterward, remove them and create a single migration for the feature.